tests/test_install.py

```diff
@@ -13,6 +13,8 @@
 
 TEST_DATA_PATH = "./testdata/test_package_specifier"
 
+SOURCE_DIR = Path(__file__).parent.parent.resolve()
```

Comment:
Remove this and used the `root` fixture instead.

Reply:
Ah nice. TIL

Reply:
Fixed.